install_config/install/advanced_install.adoc

```diff
@@ -776,15 +776,65 @@ openshift_examples_modify_imagestreams=true
 
 |`*openshift_docker_additional_registries*`
 |Specify the additional registry or registries.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
+|`*openshift_web_console_prefix*`
+|The prefix for the component images. For example, with
+ifdef::openshift-origin[]
+`openshift/origin-web-console:v3.9`, set prefix `openshift/origin-`.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/ose-web-console:v3.9`, set prefix `openshift3/ose-`.
```

Comment:
How about this:
`openshift3/ose-web-console:v3.9`, set prefix `registry.example.com/openshift3/ose-`.